lib/ed25519.js

```diff
@@ -9,16 +9,24 @@
  * https://github.com/dchest/tweetnacl-js
  */
 var forge = require('./forge');
+require('./asn1');
```

Comment:
Not a huge fan of requiring `asn1` to use ed25519 ... but I think it will take too long to get this merged if we try and engineer around it. We can save that for another day.